jwk/jwt_strategy_test.go

```diff
@@ -38,15 +38,12 @@ import (
 func TestRS256JWTStrategy(t *testing.T) {
```

Comment:
Could you add a test for this strategy using HSM?

Reply:
Test is executed using HSM when executing `make quicktest-hsm` or with CircleCI `test-hsm` job